pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor.go

```diff
@@ -70,6 +70,10 @@ type streamIngestionFrontier struct {
 	// persistedReplicatedTime stores the highwater mark of
 	// progress that is persisted in the job record.
 	persistedReplicatedTime hlc.Timestamp
+	// retainedTime is the minimum timestamp of persistedReplicatedTime
+	// and cutoverTime. It represents the earliest timestamp for which
+	// the source cluster can start garbage collection.
+	retainedTime hlc.Timestamp
```

Comment:
nit: i'd call this heartbeat time, as retainedTime already means the protected time on the destination cluster.

Reply:
done